addons/event/security/ir.model.access.csv

```diff
@@ -26,3 +26,13 @@ access_event_tag,event.tag,model_event_tag,,0,0,0,0
 access_event_tag_registration,event.tag.user,model_event_tag,event.group_event_registration_desk,1,0,0,0
 access_event_tag_user,event.tag.user,model_event_tag,event.group_event_user,1,1,1,0
 access_event_tag_manager,event.tag.manager,model_event_tag,event.group_event_manager,1,1,1,1
+access_event_question_public,event.question,model_event_question,base.group_public,1,0,0,0
```

Comment:
That's quite a security issue: here you let public / portal people have access to all questions and answers globally. Rules are defined in ``website_event`` but here you give a global access. We should probably not give any public / portal access in ``event`` and add new rules for that in ``website_event`` when publish field is added on event. Makes no sense to let public / portal read those anyway without website being installed.

Even giving access to "group_user" is probably not necessary, only registration desk people need basic access, then user people full access.